readme.md

```diff
@@ -0,0 +1,54 @@
+# Five for the Future
+
+Plugins and themes for the Five for the Future subsite: https://wordpress.org/five-for-the-future/
+
+## Contributing
+
+In order to contribute with code changes, you'll want to set up a local environment to test changes and then push the changes to a Pull Request on this Github Repository.
+
+### Initial environment setup
+
+1) Use whichever local WordPress development setup you prefer and create a new local WP site.
+2) Find the `wp-content` folder and delete it (make a backup if you have data you don't want to lose data you already have there).
+3) Fork the [five-for-the-future](https://github.com/WordPress/five-for-the-future) repository under your own Github account.
+4) Run `git clone git@github.com:WordPress/five-for-the-future.git wp-content`. Replace `WordPress/` with your Github `USERNAME/` to clone your newly forked repo instead.
+5) Ensure this newly cloned `wp-content` folder is where it should be in the WP structure.
+6) Copy over the base theme with: `svn export https://meta.svn.wordpress.org/sites/trunk/wordpress.org/public_html/wp-content/themes/pub/wporg themes/pub/wporg` (this should be run from the `wp-content` folder).
+
+### Configuring the site
+
+1) Login to your site and activate the "Five for the Future" theme and plugin.
+2) Navigate to `wp-content/themes/pub/wporg` from your CLI and run: `npm install && grunt default`
```

Comment:
Do we need this step? (I don't remember doing it, but I set things up differently)

Reply:
Oh, guess not! The parent theme uses `get_theme_file_uri()` to enqueue `css/style.css` so there is indeed no need to build it in the parent since only the child's is used.